### PR TITLE
Add clarity to intro to inventory

### PR DESCRIPTION
**Pull request by TroyHernandez**
##### SUMMARY

If the ini file is discussed, there should be a pointer to it.
I've had several people ask about this while proctoring this the last few weeks.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME

- exercises

### CODE REVIEW

File: exercises/ansible_rhel/1.1-setup/README.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Login to your control host via SSH:
...
 >
-> Replace **11.22.33.44** by the **IP** provided to you, and the **X** in student**X** by the student number provided to you.
+> Replace **11.22.33.44** by your **IP** provided to you, and the **X** in student**X** by the student number provided to you.
 
     ssh studentX@11.22.33.44
 
 > **Tip**
 >
-> Use the password provided on the workshops landing page.
+> The password is **ansible**
```
Comment (IPvSean) [inline exercises/ansible_rhel/1.1-setup/README.md]: the password changes based on every workshop, i can't accept this PR because of this single line here

Comment (TroyHernandez) [reply exercises/ansible_rhel/1.1-setup/README.md]: I didn't intend to include the 6 other commits.

Comment (TroyHernandez) [reply exercises/ansible_rhel/1.1-setup/README.md]: For example, line 22 in this commit is not grammatically correct.

Comment (IPvSean) [reply exercises/ansible_rhel/1.1-setup/README.md]: ok can you fix the PR?  This is not something I do on my end, you have to update your PR

**Pull request closed without merging**